### PR TITLE
Convert this code so that is compiles

### PR DESCRIPTION
**Pull request by JacobBallantine**
When I tested this code I had an error with line 14. The error said "wrong number of arguments for `Person.new`" So I added some more code so the code would work and it worked, for those who are testing and learning the things I added was.

### CODE REVIEW

File: syntax_and_semantics/overloading.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ We can define a `become_older` method that accepts a number indicating the years
 class Person
   @age = 0
 
+    getter :age
+    
+    def initalize(@name : String)
```
Comment (mverzilli) [inline syntax_and_semantics/overloading.md]: There's a typo here: `initalize` => `initialize`.

File: syntax_and_semantics/overloading.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ We can define a `become_older` method that accepts a number indicating the years
 class Person
   @age = 0
```
Comment (mverzilli) [inline syntax_and_semantics/overloading.md]: Can you move this line to the new constructor? I it'll be tidier.

**Comment by mverzilli** [comment]
Thanks! I left you some comments.

**Comment by JacobBallantine** [comment]
Thanks @mverzilli, I've updated the code as you've requested. 
Appreciate your feedback.

**Comment by mverzilli** [comment]
Thank you!

**Pull request merged**